boards/arduino-mkr-common/include/periph_conf.h

```diff
@@ -111,6 +111,7 @@ static const uart_conf_t uart_config[] = {
         .rx_pad   = UART_PAD_RX_3,
         .tx_pad   = UART_PAD_TX_2,
         .runstdby = 0,
+        .wakeup   = false,
```

Comment:
Maybe using a `uint8_t` `flags` field would help for future extension ? Then you can define the available flags in an `enum` and do some bitwise comparison in the driver instead of comparing to a boolean.

Reply:
I like the idea, runstdby and wakeup can already be merged into a flag variable.

Reply:
Indeed, that's even better and will same a bit of memory

Reply:
Good idea, I will make the enhancement